fuzz/README.md

```diff
@@ -0,0 +1,110 @@
+# Fuzzing
+
+`simplicity` has a fuzzing harness setup for use with honggfuzz.
+
+To run the fuzz-tests as in CI -- briefly fuzzing every target -- simply
+run
+
+    ./fuzz.sh
+
+in this directory.
+
+You need to install the Rust crate honggfuzz-rs.
+
+To build the crate you must have libunwind on your system, as well as
+libopcodes and libbfd from binutils **2.38** on your system. The most
+recently-released binutils 2.39 has changed their API in a breaking way.
```

Comment:
In 08801420c213b16598eb0d5926c7173ce6f0fd4d

This paragraph and the sentence above are confusing. The actual utility you need to install is `cargo-honggfuzz`. There is no crate called `honggfuzz-rs`; that is the name of the git repo where it comes from, if you install it from Github, which you won't if you use `cargo install`.

There's also no need to explicitly run `cargo install` because the shell script does it for you.

You also don't need libunwind or libbfd to install or build `cargo-honggfuzz`. You need these to *run* `cargo hfuzz run`, and you only need to care about this if the system-provided ones are the wrong version.